example/example_parameters_test.go

```diff
@@ -0,0 +1,93 @@
+package example_test
+
+import (
+	"fmt"
+	"github.com/dailymotion/allure-go"
+	"testing"
+)
+
+type SampleObject struct {
+	Key   string
+	Value string
+}
+
+var sampleStruct = SampleObject{
+	Key:   "some key",
+	Value: "some value",
+}
+
+var parameters = map[string]interface{}{
+	"string": "value",
+	"int":    10,
+	"struct": sampleStruct,
+}
+
+func TestAllureParameterized(t *testing.T) {
+	for i := 0; i < 5; i++ {
+		t.Run("", func(t *testing.T) {
+			allure.TestWithParameters(t,
+				"Test with parameters",
+				map[string]interface{}{
+					"counter": i,
+				}, func() {
+
+				})
+		})
+	}
+}
+
+func TestAllureParametersExample(t *testing.T) {
+	allure.TestWithParameters(t,
+		"This is a test to show allure implementation with a passing test",
+		parameters,
+		func() {
+			allure.Step(fmt.Sprintf("Number: %d", parameters["int"]), func() {})
+			allure.Step(fmt.Sprintf("String: %s", parameters["string"]), func() {})
+			allure.Step(fmt.Sprintf("Interface: %+v", parameters["struct"]), func() {})
+		})
+}
+
+func TestAllureParameterizedFailing(t *testing.T) {
+	allure.TestWithParameters(t,
+		"This is a test to show allure implementation with a passing test",
```

Comment:
Why does the name of the func mention Failing, while the description mentions passing test?

Reply:
This test function doesn't provide any substantial value anyway, removing it.